subsys/nrf_security/Kconfig.tls

```diff
@@ -8,7 +8,7 @@ menuconfig MBEDTLS_X509_LIBRARY
 	bool
 	prompt "Create mbed TLS x509 library"
 	select MBEDTLS_PK_C
-	default y if MBEDTLS_PSA_CRYPTO_C && (MBEDTLS_ECDSA_C || MBEDTLS_ECDH_C)
+	default n
```

Comment:
no default == `default n` so then we can just remove the line, as we do for most other Kconfigs, same goes for the  second Kconfig further down